### ISSUE TITLE
Is the DHT11 support working at all

### ISSUE THREAD

**Issue by acoulson2000** [dasrue/esp8266-BACnet#1]
Hi @dasrue , I am hoping to use your project to report measurements from a DHT11. I see code there for it, but was it ever functional?

**Comment by dasrue** [comment]
Hi, I've got to be honest that I have kinda abondoned this project. I just didn't have the spare time to put into it. I never bothered to get the DHT11 support working, as when I went to buy the sensor I found out the DHT11's wern't very good and that the HTU21d would be better. Hence the HTU21d code, which I believe does work.

**Comment by acoulson2000** [comment]
I could probably use the HTU21d, but I've built and flashed to the code to a Wemos D1 and it just goes into a reboot loop. Do you recall anything like that? As an experiment, I commented out everything in user_init but it still reboot loops.

**Comment by acoulson2000** [comment]
My mistake.

I switched to an ESP01 (ESP8285) and got the same thing. After some googling, I realize I was actually seeing "csum err" just before the reboot. This was due to needing to flash with "dout" mode. 

After changing the flash mode, I was still in a reboot loop, but was seeing "rf_cal[0] !=0x05,is 0xFF" just before reboot. This is a known issue that is resolved by reflashing the "init data" (esp_init_data_default.bin) as described here: https://nodemcu.readthedocs.io/en/dev/flash/ (scroll to "SDK Init Data".
